springbootsample/src/main/java/com/microsoft/applicationinsights/controller/TestController.java

```diff
@@ -0,0 +1,41 @@
+package com.microsoft.applicationinsights.controller;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+import com.microsoft.applicationinsights.telemetry.EventTelemetry;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+@RestController
+public class TestController {
+
```

Comment:
curious, why do we have empty lines here but not in the Springbootdemo class?

Reply:
Empty lines are generated by default from spring initializer. I will add it to the demo class as well.